docs/Releases.md

```diff
@@ -21,20 +23,25 @@ Homebrew release:
 If this is a major or minor release (e.g. X.0.0 or X.Y.0) then there are a few more steps:
 
 1. Before creating the tag you should delete any `odisabled` code, make any
-   `odeprecated` code `odisabled` and add any new `odeprecations` that are
-   desired.
+   `odeprecated` code `odisabled`, uncomment any `# odeprecated` code and add
+   any new `odeprecations` that are desired.
 2. Write up a release notes blog post to <https://brew.sh>
    e.g. [brew.sh#319](https://github.com/Homebrew/brew.sh/pull/319).
-   This should use `brew release-notes` as input but have the wording adjusted
-   to be more human readable and explain not just what has changed but why.
+   This should use the output from `brew release [--major|--minor]` as input but
+   have the wording adjusted to be more human readable and explain not just what has changed but why.
 3. When the release has shipped and the blog post has been merged, tweet the
    blog post as the [@MacHomebrew Twitter account](https://twitter.com/MacHomebrew)
    or tweet it yourself and retweet it with the @MacHomebrew Twitter account
    (credentials are in 1Password).
-4. Send the email to the Homebrew TinyLetter email list (credentials are in
    1Password).
-5. Consider whether to submit it to other sources e.g. Hacker News, Reddit.
-
+4. Consider whether to submit it to other sources e.g. Hacker News, Reddit.
   - Pros: gets a wider reach and user feedback
   - Cons: negative comments are common and people take this as a chance to
           complain about Homebrew (regardless of their usage)
+
+Please do not manually create a release based on older commits on the `master` branch.
+It's very hard to judge whether these have been sufficiently tested by users or if they will
+cause negative side-effects with the current state of Homebrew/homebrew-core.
+If a new branch is needed ASAP but there are things on `master` that cannot be released yet
+(e.g. new deprecations and you want to make a patch release) then revert the relevant PRs,
+follow the process above and then revert the reverted PRs to reapply them on `master`.
```

Comment:
```suggestion
follow the process above and then reapply the reverted PRs on `master`.
```

Might be a little clearer

Reply:
Kinda but using `revert` here points to the fact that you can literally revert the PR (either locally or through the GitHub UI) which is often easier than re-applying.

Reply:
👍